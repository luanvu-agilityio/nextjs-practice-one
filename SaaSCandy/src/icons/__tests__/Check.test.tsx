import { render } from '@testing-library/react';
import { CheckIcon } from '../Check';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<CheckIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
