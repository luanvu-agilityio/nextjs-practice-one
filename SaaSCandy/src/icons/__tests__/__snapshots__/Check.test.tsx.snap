// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="19"
  viewBox="0 0 18 19"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M11.6379 7.54846C11.6463 7.54833 11.6546 7.55022 11.6624 7.55334C11.6701 7.55649 11.677 7.56112 11.6829 7.56702C11.6888 7.57292 11.6934 7.5798 11.6965 7.58752C11.6997 7.59526 11.7015 7.60359 11.7014 7.61194C11.7013 7.62033 11.699 7.62869 11.6956 7.63635C11.6922 7.64392 11.6879 7.65119 11.6819 7.65686L11.675 7.66272L7.91919 11.4186C7.90755 11.4302 7.89168 11.437 7.87524 11.4371C7.85868 11.4371 7.84204 11.4303 7.83032 11.4186H7.8313L6.34106 9.92834C6.33174 9.917 6.32625 9.90306 6.32642 9.88831C6.3267 9.87211 6.33352 9.85581 6.34497 9.84436C6.35639 9.83313 6.37192 9.82706 6.38794 9.82678C6.39602 9.82666 6.40397 9.82784 6.41138 9.83069L6.43188 9.84338L7.87524 11.2867L11.5872 7.57483L11.593 7.56799C11.5987 7.56194 11.606 7.55771 11.6135 7.55432C11.6212 7.55091 11.6295 7.54861 11.6379 7.54846Z"
    fill="black"
    stroke="#37A05B"
  />
  <path
    d="M9 1.9375C13.1766 1.9375 16.5625 5.32339 16.5625 9.5C16.5625 13.6766 13.1766 17.0625 9 17.0625C4.82339 17.0625 1.4375 13.6766 1.4375 9.5C1.4375 5.32339 4.82339 1.9375 9 1.9375ZM9 2.0625C7.02745 2.0625 5.13601 2.84641 3.74121 4.24121C2.34641 5.63601 1.5625 7.52745 1.5625 9.5C1.5625 11.4725 2.34641 13.364 3.74121 14.7588C5.13601 16.1536 7.02745 16.9375 9 16.9375C10.9725 16.9375 12.864 16.1536 14.2588 14.7588C15.6536 13.364 16.4375 11.4725 16.4375 9.5C16.4375 7.52745 15.6536 5.63601 14.2588 4.24121C12.864 2.84641 10.9725 2.0625 9 2.0625Z"
    fill="black"
    stroke="#37A05B"
  />
</svg>
`;
